### PR TITLE
Move to data version 8.0.5

### PR DESCRIPTION
**Pull request by rheacangeo**
This is a small update to the regression data due to changes made in the fortran branch before merging to master in https://github.com/ai2cm/fv3gfs-fortran/pull/267

### CODE REVIEW

**Comment by jdahm** [comment]
launch jenkins

**Comment by jdahm** [comment]
launch jenkins

**Review by elynnwu — APPROVED** [review]

File: pace-util/pace/util/constants.py [hunk 1]
```diff
@@ -31,24 +31,42 @@
...
+    RDGAS = 287.05  # Gas constant for dry air [J/kg/deg] # 287.04
+    RVGAS = 461.50  # Gas constant for water vapor [J/kg/deg]
+    HLV = 2.5e6  # Latent heat of evaporation [J/kg]
+    HLF = 3.3358e5  # Latent heat of fusion [J/kg]  # 3.34e5
+    CP_AIR = 1004.6
+    KAPPA = RDGAS / CP_AIR  # Specific heat capacity of dry air at
+    TFREEZE = 273.15
+else:
+    RADIUS = 6371.0e3  # Radius of the Earth [m] #6371.0e3
+    PI = 3.14159265358979323846  # 3.14159265358979323846
```
Comment (jdahm) [inline pace-util/pace/util/constants.py:54]: Woah it's an alternate reality!

**Comment by elynnwu** [comment]
launch jenkins

**Comment by elynnwu** [comment]
launch jenkins

**Pull request merged**